lib/v2/xys/new.js

```diff
@@ -0,0 +1,70 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'http://www.xys.org';
+    const currentUrl = `${rootUrl}/new.html`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+        responseType: 'buffer',
+    });
+
+    // 转码
+    const data = iconv.decode(response.data, 'gb2312');
+
+    const $ = cheerio.load(data);
+
+    let items = $('li a')
+        .slice(0, 10)
```

Comment:
```suggestion
        .slice(0, ctx.query.limit ? parseInt(ctx.query.limit) : 30)
```
Looks like 10 is not enough since there're more than 10 items on 23rd.

You may need to skip the first 4 items as well.

Reply:
thanks, fixed as suggested!